src/app/app.routes.ts

```diff
@@ -26,6 +26,7 @@ export interface RouteData {
   isPublic?: boolean;
 
   dialogToken?: DialogComponentToken;
+  component?: any;
```

Comment:
Does this need to be `any` type? I feel like this should match the type that is allowed by the CDK Dialog `open` method. Also the `Component` type included at the top is currently unused in this file.